packages/xrpl/test/integration/transactions/delegateSet.test.ts

```diff
@@ -90,6 +93,46 @@ describe('DelegateSet', function () {
       assert.equal(delegateLedgerEntry.Authorize, bob.address)
       assert.equal(delegateLedgerEntry.Permissions.length, 2)
 
+      // Verify Delegate ledger data
+      const grantedPermissions = new Set<string>()
```

Comment:
Why is this check needed? Seems pretty unnecessary if it shows up in `account_objects`, which is also going to be much more efficient to run (since you don't have to loop through _all_ ledger objects, which is a nontrivial amount during integration test runs)

Reply:
That's true. I'll remove this change.